Features chapter5

IIKUYY opened the pull request:
## what

- Agregar el resumen del capitulo 4 junto con sus ejemplos
- Agregar vínculos a los menús

## why

- Continuar con las tareas dadas
- Mantener el uso correcto de TOC de navegación

## references
- http://diveintohtml5.info/index.html

IIKUYY committed:
Feat: Actualizar TOC de navgación

## what

- Agregar vínculos a los menús

## why

- Mantener el uso correcto de TOC de navegación

## references
-

IIKUYY committed:
Docs: Agregar resumen del capitulo 5

## what

- Agregar el resumen del capitulo 4 junto con sus ejemplos

## why

- Continuar con las tareas dadas

## references
- http://diveintohtml5.info/index.html

semanticmx commented on Ch5/Ejemplo/index.html:
```diff
@@ -0,0 +1,38 @@
+            height="240" 
+            preload 
+            controls>
+                <source src="pr6.webm" type="video/webm; codecs=vp8,vorbis"/>
```
los atributos de esta etiqueta también deben ir en 1 sola columna.

semanticmx commented on Ch5/Ejemplo/index.html:
```diff
@@ -0,0 +1,38 @@
+                <source src="pr6.webm" type="video/webm; codecs=vp8,vorbis"/>
+                <source src="pr6.ogv" type="video/ogg; codecs=theora,vorbis"/>
+                <source src="pr6.mp4"/>
+                <object width="320" height="240" type="application/x-shockwave-flash" data="http://releases.flowplayer.org/swf/flowplayer-3.2.1.swf">
```
igual acá, una sola columna

semanticmx commented on Ch5/Ejemplo/index.html:
```diff
@@ -0,0 +1,38 @@
+                    <param name="allowfullscreen" value="true"/>
+                    <param name="flashvars" value="config={'clip':{'url':'http://diveintohtml5.info/i/pr6.mp4','autoPlay':false,'autoBuffering':true}}"/>
+                    <p>
+                        Download video as <a href="i/pr6.mp4">MP4</a>
```
la etiqueta <a> va en una línea a parte.

semanticmx reviewed (changes requested):
también puedes agregar screenshots de algún video

IIKUYY committed:
Chores: Realizar cambios requeridos

## what

- Corregir la identación
- Agregar capturas de pantalla junto a la carpeta assets
- Corregir semántica de HTML

## why

- Arreglar la semántica de los archivos
- Mejorar calidad de lectura del repositorio

## references
-

IIKUYY commented:
Cambios realizados conforme a lo requerido

semanticmx reviewed (approved):
muy bien!
